From 30afe3adc43891447e3f76e189e861cf571df1b8 Mon Sep 17 00:00:00 2001
From: Timotheus Pokorra <tp@tbits.net>
Date: Wed, 9 Mar 2016 15:52:42 +0100
Subject: [PATCH] remove stray line that would reset the schema_files list, and
 try to install only the chwala mysql files. fixes
 https://issues.kolab.org/show_bug.cgi?id=5355 error: table roundcube.system
 does not exist

Differential Revision: https://git.kolab.org/D97
---
 pykolab/setup/setup_roundcube.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pykolab/setup/setup_roundcube.py b/pykolab/setup/setup_roundcube.py
index 5bbeda0..82073cc 100644
--- a/pykolab/setup/setup_roundcube.py
+++ b/pykolab/setup/setup_roundcube.py
@@ -193,7 +193,6 @@ def execute(*args, **kw):
                 if not schema_filepath in schema_files:
                     schema_files.append(schema_filepath)
 
-    schema_files = []
     for root, directories, filenames in os.walk('/usr/share/doc/'):
         directories.sort()
         for directory in directories:
-- 
1.8.3.1

